subprojects/architecture-test/src/test/java/org/gradle/architecture/test/ArchUnitFixture.java

```diff
@@ -96,6 +96,17 @@ public boolean test(Collection<T> input) {
         };
     }
 
+    static ArchCondition<JavaClass> beAbstract() {
+        return new ArchCondition<JavaClass>("verify that classes are abstract") {
```

Comment:
```suggestion
        return new ArchCondition<JavaClass>("be abstract") {
```

This should be the same name as the method, so the phrase is reflected in the error message and reads well.